feature-libs/cart/main/core/facade/active-cart.service.ts

```diff
@@ -213,50 +182,50 @@ export class ActiveCartService implements ActiveCartFacade, OnDestroy {
    * Returns true when cart is stable (not loading and not pending processes on cart)
    */
   isStable(): Observable<boolean> {
-    // Debounce is used here, to avoid flickering when we switch between different cart entities.
-    // For example during `addEntry` method. We might try to load current cart, so `current cart will be then active id.
-    // After load fails we might create new cart so we switch to `temp-${uuid}` cart entity used when creating cart.
-    // At the end we finally switch to cart `code` for cart id. Between those switches cart `isStable` function should not flicker.
```

Comment:
Could thee flickering still be an issue if we remove the debounce?

Reply:
It calls `this.multiCartService.isStable(cartId))`, which already has `debounce((isStable) => (isStable ? timer(0) : EMPTY)), distinctUntilChanged()`